content/docs/user-guide/data-management/index.md

```diff
@@ -0,0 +1,97 @@
+# Data Management with DVC
+
+DVC helps you manage and share arbitrarily large files, datasets, and ML models
```

Comment:
It's too abstract unfortunately. We need to talk about value prop here, we need to mention clouds first (this is major still).

How about:

Data is a first-class citizen in data science workflow ... DVC organizes unstructured data (tools for it almost doesn;t exist)... DVC data management layer guarantees reproducibility of your models, infrastructure for caching objects to avoid network .... , enables automation like CI/CD ... etc ..

we can think about this more.

Reply:
> We need to talk about value prop here

Not sure TBH. According to our current [framework](https://www.notion.so/iterative/Framework-7e86e484e50e42529df3dbfe32b4e987), we deliver the value in use cases and esp. in Get Started.

> summary of the value prop is Versioning + Sharing?

@dberenbaum pretty much what we cover in https://dvc.org/doc/start/data-management/data-versioning, reinforcing my previous comment.

So far my approach to this guide is to explain the parts that make up this feature set. For that reason I don't even get into versioning much (until the end, but we may even want to make a separate page for that, see https://github.com/iterative/dvc.org/pull/4042#pullrequestreview-1150329841 below).

Reply:
> DVC organizes unstructured data

How?

Well, for now I flattened & shortened the index page (except for the Versioning section) but it's still about the parts of DVC that enable its data management approach (local, external, remote locations // workspace, cache, remotes). PTAL

Reply:
From chat with @shcheklein (on [index page](https://dvc-org-guide-data-mgmt-lfnbat.herokuapp.com/doc/user-guide/data-management))
- Still too abstract (text) and maybe too technical (diagram)
- Focus on (changed) workflows (e.g. from `aws s3 cp` to `dvc get`)
- Understand why its important to "pay this price" (codify, separate storage, go through git repo)
- Benefits (similar to use cases); indirection: DVC orgs objects into dirs, you deal with project-specific refs.
- Remote storage may be optional but it's kind of defaults

Reply:
OK, I started addressing all this but the doc is getting a little long, may need another review iteration to get it back into the right track. [PTAL](https://dvc-org-guide-data-mgmt-lfnbat.herokuapp.com/doc/user-guide/data-management)